pkg/blockatlas/tx.go

```diff
@@ -149,14 +147,14 @@ type ContractCall struct {
 
 // AnyAction describes all other types
 type AnyAction struct {
-	Coin     uint   `json:"coin"`
-	Title    string `json:"title"`
-	Key      string `json:"key"`
-	TokenID  string `json:"token_id"`
-	Name     string `json:"name"`
-	Symbol   string `json:"symbol"`
-	Decimals uint   `json:"decimals"`
-	Value    Amount `json:"value"`
+	Coin     uint        `json:"coin"`
+	Title    string      `json:"title"`
+	Key      KeyType     `json:"key"`
+	TokenID  string      `json:"token_id,omitempty"`
+	Name     string      `json:"name,omitempty"`
```

Comment:
iOS requires those values, otherwise it won't be able to parse it

<img width="716" alt="Screen Shot 2019-11-17 at 5 59 55 PM" src="https://user-images.githubusercontent.com/1641795/69019119-11e7b980-0964-11ea-940b-254ad54e25ca.png">